src/utils/proguard_upload.rs

```diff
@@ -58,16 +58,12 @@ impl ChunkedMapping {
 
 impl<'a> From<&'a ChunkedMapping> for ChunkedDifRequest<'a> {
     fn from(value: &'a ChunkedMapping) -> Self {
-        ChunkedDifRequest {
-            name: &value.file_name,
-            debug_id: None,
-            chunks: &value.chunk_hashes,
-        }
+        ChunkedDifRequest::new(&value.file_name, &value.chunk_hashes, value.hash)
     }
 }
 
-fn to_assemble(chunked: &ChunkedMapping) -> (Digest, ChunkedDifRequest<'_>) {
-    (chunked.hash, chunked.into())
+fn to_assemble(chunked: &ChunkedMapping) -> ChunkedDifRequest<'_> {
```

Comment:
Is this function still necessary?

Reply:
I suppose we can just call `chunked.into()` directly